### PR TITLE
:wrench: Add `tflint` to terraform feature

### PR DESCRIPTION
**Pull request by Gary-H9**
Resolves https://github.com/ministryofjustice/.devcontainer/issues/118

### CODE REVIEW

File: features/src/terraform/CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.1.0] - 2024-05-13
+
+### Changed
+
+- Added `tflint`
+
+
```
Comment (jacobwoffenden) [inline features/src/terraform/CHANGELOG.md:17]:
```suggestion
```

File: features/src/terraform/devcontainer-feature.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
 {
   "id": "terraform",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "name": "Terraform",
   "documentationURL": "https://github.com/ministryofjustice/.devcontainer/tree/main/features/src/terraform",
-  "description": "Installs the Terraform CLI",
+  "description": "Installs the Terraform CLI + tflint",
```
Comment (jacobwoffenden) [inline features/src/terraform/devcontainer-feature.json:6]:
```suggestion
  "description": "Installs the Terraform CLI and tflint",
```

File: features/src/terraform/devcontainer-feature.json
Comment (jacobwoffenden) [inline features/src/terraform/devcontainer-feature.json:1]: This file is missing the `tflintVersion` option

File: features/src/terraform/src/home/vscode/.devcontainer/featurerc.d/tflint-cli.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+complete -o nospace -C /usr/local/bin/tflint tflint
```
Comment (jacobwoffenden) [inline features/src/terraform/src/home/vscode/.devcontainer/featurerc.d/tflint-cli.sh:3]: is this valid?

**Review by jacobwoffenden — CHANGES_REQUESTED** [review]
Added some comments, and missing tests

File: features/src/terraform/CHANGELOG.md (hunk 1 above)
Comment (Gary-H9) [reply features/src/terraform/CHANGELOG.md]: Done.

File: features/src/terraform/devcontainer-feature.json
Comment (Gary-H9) [reply features/src/terraform/devcontainer-feature.json:1]: Done.

File: features/src/terraform/src/home/vscode/.devcontainer/featurerc.d/tflint-cli.sh (hunk 3 above)
Comment (Gary-H9) [reply features/src/terraform/src/home/vscode/.devcontainer/featurerc.d/tflint-cli.sh]: Removed entirely.

File: features/src/terraform/devcontainer-feature.json (hunk 2 above)
Comment (Gary-H9) [reply features/src/terraform/devcontainer-feature.json]: Edited.

File: features/src/terraform/CHANGELOG.md [hunk 4]
```diff
@@ -8,6 +8,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.1.0] - 2024-05-13
```
Comment (jacobwoffenden) [inline features/src/terraform/CHANGELOG.md:11]:
```suggestion
## [1.1.0] - 2024-08-22
```

**Review by jacobwoffenden — CHANGES_REQUESTED** [review]

**Review by jacobwoffenden — APPROVED** [review]

**Pull request merged**